frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/components/UploadStatus/style.ts

```diff
@@ -35,23 +36,12 @@ export const DatasetStatusTag = styled.div`
   align-self: flex-start;
   width: fit-content;
   padding: ${PT_GRID_SIZE_PX}px;
-  margin: 0 auto;
```

Comment:
Also removing centering the status tag, because the mock shows the status to be left aligned:

BEFORE:
<img width="208" alt="Screen Shot 2021-01-20 at 11 00 33 AM" src="https://user-images.githubusercontent.com/6309723/105222035-c522e400-5b0e-11eb-9966-038b4fb6461a.png">


AFTER:
<img width="422" alt="Screen Shot 2021-01-20 at 10 59 17 AM" src="https://user-images.githubusercontent.com/6309723/105221950-a7ee1580-5b0e-11eb-8733-3ee6a59834d8.png">

Mock:
<img width="357" alt="Screen Shot 2021-01-20 at 11 01 13 AM" src="https://user-images.githubusercontent.com/6309723/105222081-d79d1d80-5b0e-11eb-8156-9d90a40a2b74.png">